definitions/checks/disk_speed_minimal.rb

```diff
@@ -0,0 +1,9 @@
+class Checks::DiskSpeedMinimal < ForemanMaintain::Check
+  for_feature :disk_io
+  description 'Check for recommended disk speed of pulp, mongodb, pgsql dir.'
+  tags :pre_upgrade
+
+  def run
+    assert(feature(:disk_io).pass?, "Disk speed should be atleast #{MIN_SPEED_MBPS} MB/sec")
```

Comment:
The info message sould be more specfic about what subdirs don't match the requirement and what's the current value for the particular disk